libs/d3d12core/main.c

```diff
@@ -211,7 +213,42 @@ static char *openvr_device_extensions(DXGI_ADAPTER_DESC *desc)
     return openvr_device_extensions;
 }
 
-static uint32_t parse_openvr_extension_list(char *extension_str, char **extension_list)
+static char *openxr_vulkan_extensions(bool device_extensions)
+{
+
+    PFN___wineopenxr_GetVulkanExtensions get_extensions = NULL;
+    const char *sym = device_extensions ? "__wineopenxr_GetVulkanDeviceExtensions"
+                                        : "__wineopenxr_GetVulkanInstanceExtensions";
+    uint32_t len;
+    char *ret = NULL;
+    if (!wineopenxr_module)
```

Comment:
This is not thread safe. Should be done in load_d3d12core as part of the pthread_once work.